packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/index.tsx

```diff
@@ -125,11 +128,17 @@ const List = ({ isAlternative, quotes }: Props) => {
                                 {fiatCurrency}
                             </Text>
                             <StyledIcon icon="ARROW_RIGHT_LONG" />
-                            {wantCrypto && (
-                                <Receive>{formatCryptoAmount(Number(cryptoStringAmount))}</Receive>
-                            )}
                             <StyledCoinLogo size={21} symbol={account.symbol} />
-                            <Crypto>{receiveCurrency}</Crypto>
+                            {wantCrypto ? (
```

Comment:
What's happening here? 👀

Reply:
You either want an amount in crypto or in fiat. This distinguishes different use cases. If you choose fiat, only crypto icon and symbol are shown.

Reply:
got it!